src/components/TodoList.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onRemove: (todoId: number) => void
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onRemove,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map((todo) => (
+        <>
+          <div
+            key={todo.id}
+            className={classNames(
+              'todo',
+              { completed: todo.completed },
+            )}
+          >
+            <label
+              className="todo__status-label"
+            >
+              <input
+                type="checkbox"
+                className="todo__status"
+                checked={todo.completed}
```

Comment:
> You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.